airflow/models/baseoperator.py

```diff
@@ -1662,10 +1660,11 @@ def defer(
         raise TaskDeferred(trigger=trigger, method_name=method_name, kwargs=kwargs, timeout=timeout)
 
 
-Chainable = Union[BaseOperator, "XComArg", EdgeModifier, "TaskGroup"]
+# TODO: Deprecate for Airflow 3.0
+Chainable = Union[DependencyMixin, Sequence[DependencyMixin]]
```

Comment:
Maybe it’s actually a good idea to keep using this name…? It’s unfortunately not super easy to deprecate a type alias.

Reply:
I don't think this alias is used anywhere beyond the usage a few lines below.